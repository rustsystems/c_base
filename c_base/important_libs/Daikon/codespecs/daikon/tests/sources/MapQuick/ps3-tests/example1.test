# Sample Testing file

# Create a graph
CreateGraph graph1

# Add a pair of nodes
CreateNode n1 5
CreateNode n2 5
AddNode graph1 n1
AddNode graph1 n2

# Add an edge
AddEdge graph1 n1 n2

# Print some stuff
ListNodes graph1
ListChildren graph1 n1
ListChildren graph1 n2

# Find a path
FindPath graph1 n1 -> n2

