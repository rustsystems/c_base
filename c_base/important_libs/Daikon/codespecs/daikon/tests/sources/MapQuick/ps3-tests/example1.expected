# Sample Testing file

# Create a graph
created graph graph1

# Add a pair of nodes
created node n1 with cost 5
created node n2 with cost 5
added node n1 to graph1
added node n2 to graph1

# Add an edge
added edge from n1 to n2 in graph1

# Print some stuff
graph1 contains: n1 n2
the children of n1 in graph1 are: n2
the children of n2 in graph1 are: 

# Find a path
shortest path in graph1: n1 n2

